### ISSUE TITLE
Error: No IMDB on PTP

### ISSUE THREAD

**Issue by KingMak3r** [L4GSP1KE/Upload-Assistant#295]
I noticed that in a certain situation, when uploading a release to PTP that does not have an IMDB page (i.e. IMDB value set to 0), the upload fails. 

I know this is a pretty unique case but I thought you would like to  know.

**Comment by swannie-eire** [comment]
Isn't IMDB page a minimum requirement of PTP?

You can always create an IMDB page once you create an account.

**Comment by KingMak3r** [comment]
> Isn't IMDB page a minimum requirement of PTP?
> 
> You can always create an IMDB page once you create an account.

That is correct, IMDB is a minimum requirement on PTP and a torrent can be uploaded without IMDB or TMDB links. However, I noticed that the script consistently fails in theese instances. That is when uploading a unique title that has a TMDB page but no IMDB page.

About IMDB pages, I wasnt aware you could create new pages. I've created numerous TMDB pages but never an IMDB page.
